utils/tile_splitter/grid.py

```diff
@@ -397,7 +405,7 @@ def row(self, y):
         for _ in range(y):
             right_of_row = right_of_row.neighboors[SOUTH]
 
-    def split_tile(self, tile, tile_type_pkeys, split_direction):
+    def split_tile(self, tile, tile_type_pkeys, split_direction, split_map):
```

Comment:
I'd add split_map to the docstring and explain what it is.

Reply:
Agreed

Reply:
Done